config/config.go

```diff
@@ -537,10 +535,8 @@ type TiKVClient struct {
 	ExternalConsistency bool `toml:"external-consistency" json:"external-consistency"`
```

Comment:
What about also moving it as a session variable? So we can set external consistency only for some sessions.